Python/bytecodes.c

```diff
@@ -2054,20 +2054,19 @@ dummy_func(
             #endif  /* ENABLE_SPECIALIZATION */
         }
 
-        op(_LOAD_ATTR, (owner -- attr, self_or_null if (oparg & 1))) {
+        op(_LOAD_ATTR, (owner -- attr[1], self_or_null[1] if (oparg & 1))) {
```

Comment:
```suggestion
        op(_LOAD_ATTR, (owner -- attr, self_or_null if (oparg & 1))) {
```

You don't need the arrays. The code generator should spill the values to the in-memory stack if necessary.